### PR TITLE
[U1] Frontend: Login - Basic components

### PR DESCRIPTION
**Pull request by germangelv**
## Summary
- Added the basic components of the login view

## Trello Card
https://trello.com/c/d5MMUFtV/

## Screenshot
Screenshot_1565027670

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Override
+    public void init() {
+        EditText vEmailAccount = requireActivity().findViewById(R.id.vEmailAccount);
+        EditText vPassword = requireActivity().findViewById(R.id.vPassword);
+        Button vLogin = requireActivity().findViewById(R.id.vLogInButton);
+        Button vSignup = requireActivity().findViewById(R.id.vSignUpButton);
+
+        vLogin.setOnClickListener(view -> {
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: You should look in the methods you can override, there should be a setListeners in wolmo. All onClickListener should be inside that method

Comment (igsosa92) [reply app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java]: There are a few things to mention about this. I think the best is to talk about it. Please write me after so we can have a call

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <RelativeLayout
```
[the remark this answers is not in the document]
Comment (germangelv) [reply app/src/main/res/layout/fragment_login.xml]: I correct the gradient color and the relative location of the logo
image

**Pull request merged**